settings.py

```diff
@@ -415,8 +415,9 @@ def get_locales():
 )
 
 STATICFILES_FINDERS = (
-    'django.contrib.staticfiles.finders.AppDirectoriesFinder',
-    'django.contrib.staticfiles.finders.FileSystemFinder',
+    'pipeline.finders.AppDirectoriesFinder',
+    'pipeline.finders.FileSystemFinder',
```

Comment:
It turns out this finder tries ignores files in a `build` folder, effectively making our system moot since we use "build" as the name of the folder we generate the built files into